cypress/integration/projectSettings/project_settings.ts

```diff
@@ -95,191 +93,211 @@ describe("Clicking on The Project Select Dropdown", () => {
-      ).should("not.exist");
+    it("Does not show a 'Default to Repo' button on page", () => {
+      cy.dataCy("default-to-repo-button").should("not.exist");
     });
 
-    it("Enabling commit queue shows hidden inputs and error banner", () => {
-      const countCQFields = (count: number) => {
-        cy.dataCy("cq-card").children().should("have.length", count);
-      };
-
-      countCQFields(2);
-      cy.dataCy("cq-enabled-radio-box").children().first().click();
-      countCQFields(4);
-
-      cy.dataCy("error-banner")
-        .contains(
-          "A Commit Queue Patch Definition must be specified for this feature to run."
-        )
-        .should("exist");
+    it("Does not show a 'Move to New Repo' button on page", () => {
+      cy.dataCy("move-repo-button").should("not.exist");
     });
 
-    it("Shows merge method only if merge queue is Evergreen", () => {
-      const selectId = "merge-method-select";
-
-      // Hides merge method for GitHub.
-      cy.getInputByLabel("GitHub").check({ force: true });
-      cy.dataCy(selectId).should("not.exist");
-
-      // Shows merge method for Evergreen.
-      cy.getInputByLabel("Evergreen").check({ force: true });
-      cy.dataCy(selectId).should("exist");
-      cy.get(`button[name=${selectId}]`).click();
-      cy.get(`#${selectId}-menu`).children().should("have.length", 3);
-      cy.get(`#${selectId}-menu`).children().first().click();
+    it("Does not show an Attach/Detach to Repo button on page", () => {
+      cy.dataCy("attach-repo-button").should("not.exist");
     });
 
-    it("Does not show override buttons for commit queue patch definitions", () => {
-      cy.dataCy("cq-override-radio-box").should("not.exist");
+    it("Does not show a 'Go to repo settings' link on page", () => {
+      cy.dataCy("attached-repo-link").should("not.exist");
     });
-
-    it("Updates the commit queue message", () => {
-      cy.dataCy("cq-message-input").type("Repo message");
+    it("Inputting a display name then clicking save shows a success toast", () => {
+      cy.dataCy("display-name-input").type("evg");
+      clickSave();
+      cy.validateToast("success", "Successfully updated repo");
     });
+  });
 
-    it("Disables save button because Commit Queue definition is missing", () => {
+  describe("GitHub/Commit Queue page", () => {
+    beforeEach(() => {
+      cy.dataCy("navitem-github-commitqueue").click();
       saveButtonEnabled(false);
     });
+    describe("GitHub section", () => {
+      it("Shows an error banner when Commit Checks are enabled and hides it when Commit Checks are disabled", () => {
+        cy.dataCy("github-checks-enabled-radio-box")
+          .contains("label", "Enabled")
+          .click();
+        cy.dataCy("error-banner")
+          .contains(
+            "A Commit Check Definition must be specified for this feature to run."
+          )
+          .as("errorBanner")
+          .should("be.visible");
```

Comment:
```suggestion
        cy.dataCy("error-banner")
          .contains(
            "A Commit Check Definition must be specified for this feature to run."
          )
          .as("errorBanner")

        cy.get("@errorBanner")
          .should("be.visible");
```

I think this reads better and breaks up the variable assignment from the assertion.